lib/src/core/toastification_config_provider.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/widgets.dart';
+import 'package:toastification/src/core/toastification_config.dart';
+
+class ToastificationConfigProvider extends InheritedWidget {
```

Comment:
Can you move this widget to the following path?

`/src/widget/toastification_config_provider.dart`